dev_utils/docker-compose.yml

```
@@ -91,6 +95,15 @@ services:
       - AWS_REGION=us-east-1
       - AWS_READYPATH=/minio/health/ready
       - AWS_CACERT=/certs/ca.crt
+      - DB_HOST=db
+      - DB_PORT=5432
+      - DB_USER=lega_in
+      - DB_PASSWORD=inpass
```

```suggestion
      - DB_PASSWORD=lega_in
```
Just to keep them the same in every repo